yt_dlp/extractor/motherless.py

```diff
@@ -148,102 +177,110 @@ def _real_extract(self, url):
         },
-        'playlist_mincount': 2500,
+        'playlist_mincount': 2040,
     }]
 
     @classmethod
     def suitable(cls, url):
-        return (False if MotherlessIE.suitable(url)
-                else super(MotherlessGroupIE, cls).suitable(url))
+        return False if MotherlessIE.suitable(url) else super(MotherlessGroupIE, cls).suitable(url)
 
-    def _extract_entries(self, webpage, base):
-        entries = []
-        for mobj in re.finditer(
-                r'href="(?P<href>/[^"]+)"[^>]*>(?:\s*<img[^>]+alt="[^-]+-\s(?P<title>[^"]+)")?',
-                webpage):
-            video_url = compat_urlparse.urljoin(base, mobj.group('href'))
-            if not MotherlessIE.suitable(video_url):
-                continue
-            video_id = MotherlessIE._match_id(video_url)
-            title = mobj.group('title')
-            entries.append(self.url_result(
-                video_url, ie=MotherlessIE.ie_key(), video_id=video_id,
-                video_title=title))
-        # Alternative fallback
-        if not entries:
-            entries = [
-                self.url_result(
-                    compat_urlparse.urljoin(base, '/' + entry_id),
-                    ie=MotherlessIE.ie_key(), video_id=entry_id)
-                for entry_id in orderedSet(re.findall(
-                    r'data-codename=["\']([A-Z0-9]+)', webpage))]
-        return entries
+    def _correct_path(self, url, item_id):
+        return urllib.parse.urljoin(url, f'/gv/{item_id}')
 
-    def _real_extract(self, url):
-        group_id = self._match_id(url)
-        page_url = compat_urlparse.urljoin(url, '/gv/%s' % group_id)
-        webpage = self._download_webpage(page_url, group_id)
-        title = self._search_regex(
-            r'<title>([\w\s]+\w)\s+-', webpage, 'title', fatal=False)
-        description = self._html_search_meta(
-            'description', webpage, fatal=False)
-        page_count = str_to_int(self._search_regex(
-            r'(\d+)\s*</(?:a|span)>\s*<(?:a|span)[^>]+(?:>\s*NEXT|\brel\s*=\s*["\']?next)\b',
-            webpage, 'page_count', default=0))
-        if not page_count:
-            message = self._search_regex(
-                r'''class\s*=\s*['"]error-page\b[^>]*>\s*<p[^>]*>\s*(?P<error_msg>[^<]+)(?<=\S)\s*''',
-                webpage, 'error_msg', default=None) or 'This group has no videos.'
-            self.report_warning(message, group_id)
-            page_count = 1
-        PAGE_SIZE = 80
 
-        def _get_page(idx):
-            if idx > 0:
-                webpage = self._download_webpage(
-                    page_url, group_id, query={'page': idx + 1},
-                    note='Downloading page %d/%d' % (idx + 1, page_count)
-                )
-            for entry in self._extract_entries(webpage, url):
-                yield entry
+class MotherlessGalleryIE(MotherlessPaginatedIE):
+    _VALID_URL = r'https?://(?:www\.)?motherless\.com/G[VIG]?(?P<id>[A-F0-9]+)'
```

Comment:
same as above, instead of redefining `suitable`, I think this would work:
```suggestion
    _VALID_URL = r'https?://(?:www\.)?motherless\.com/G[VIG]?(?P<id>[A-F0-9]+)/?(?:$|[#?])'
```